### ISSUE TITLE
UserWarning:  `max_length` is ignored when `padding`=`True`.

### ISSUE THREAD

**Issue by MitraMitraMitra** [MaartenGr/BERTopic#268]
Sometimes, when I run
```
topic_model_descriptions = BERTopic(language="multilingual")
topics_descriptions, probs_descriptions = topic_model_descriptions.fit_transform(descriptions)
```
I get a message saying
```
UserWarning:
`max_length` is ignored when `padding`=`True`.
```
Should I worry about it? Could it be caused by low memory? The model looks good.

**Comment by MaartenGr** [comment]
This should not be an issue in practice. Which version of BERTopic and SentenceTransformers are you using? If you want to remove the user warning, perhaps updating will resolve it.

**Comment by MaartenGr** [comment]
Due to inactivity, I'll be closing this issue for now. However, if you are still experiencing this issue or want to discuss it further, please let me know and I'll reopen it.
